docs/src/pages/guides/deploy.md

```diff
@@ -312,6 +312,24 @@ Follow the wizard started by the extension to give your app a name, choose a fra
 
 The action will work to deploy your app (watch its progress in your repo's Actions tab) and, when successfully completed, you can view your app in the address provided in the extension's progress window by clicking the 'Browse Website' button that appears when the GitHub action has run.
 
+## Cloudflare Pages
+
+You can deploy your Astro project on [Cloudflare Pages](https://pages.cloudflare.com). You need:
+
+- A Cloudflare account. If you don”t already have one, you can create a free Cloudflare account during the process.
+- Your app code pushed to a [GitHub](https://github.com) repository.
+
+Then, set up a new project on Cloudflare Pages. **Important**: you must specify a [Node version that’s compatible with Astro](https://docs.astro.build/installation#prerequisites), since the Cloudflare Pages default version may not work. 
```

Comment:
Could you add a recommendation of how to specify the node version? Does adding an .nvmrc file to your project work?

Reply:
Probably best to remove that "Important" item since I explained it in the "Environment Variable" setting. Will do so. Thanks.

Reply:
Have now changed in "my" version. Do I need to repeat the PR?